src/components/panel/panel.scss

```diff
@@ -31,7 +26,7 @@
 
 .md-panel {
   opacity: 0;
-  position: fixed;
+  position: relative;
```

Comment:
This is the change that breaks `md-tooltip`. With this, it causes the `height` style to no longer have any effect

https://github.com/angular/material/blob/e3f977214010aa8217c79dfc732e504d83a654bf/src/components/tooltip/tooltip.scss#L20

I can work around this manually by adding `padding: 5px 8px 6px;` to the `md-tooltip`, but that's not something that I can programatically map `$tooltip-height-sm` or `$tooltip-height-lg` into.

Reply:
Disclaimer that I was testing this against the docs site without any of these changes so the effect might not be the same.

I think the problem is that `md-tooltip` is a custom element which defaults to `display: inline`. It should work like before if you set `display: block` or `display: inline-block` on `.md-tooltip`.